### PR TITLE
[16.0][IMP] crm_industry: Add secondary industry

### PR DESCRIPTION
**Pull request by joanbaldo-s73**
Add `secondary_industry_ids` to crm views with optional show

### CODE REVIEW

**Review by ferran-S73 — APPROVED** [review]

**Comment by ferran-S73** [comment]
@OCA/crm-sales-marketing-maintainers could someone review pls?

File: crm_industry/views/crm_lead_view.xml [hunk 1]
```diff
@@ -28,6 +28,11 @@
         <field name="arch" type="xml">
             <field name="partner_id" position="after">
                 <field name="industry_id" />
+                <field
+                    name="secondary_industry_ids"
+                    widget="many2many_tags"
+                    optional="show"
```
Comment (pedrobaeza) [inline crm_industry/views/crm_lead_view.xml]:
```suggestion
                    optional="hide"
```

File: crm_industry/views/crm_lead_view.xml [hunk 2]
```diff
@@ -56,6 +61,11 @@
...
```
Comment (pedrobaeza) [inline crm_industry/views/crm_lead_view.xml]:
```suggestion
                    optional="hide"
```

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
This field is not so important

File: crm_industry/views/crm_lead_view.xml (hunk 2 above)
Comment (ferran-S73) [reply crm_industry/views/crm_lead_view.xml]: @pedrobaeza change applied + rebased

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**